### PR TITLE
Fix types in the callback interfaces examples

### PR DESCRIPTION
**Pull request by quad**
I got thrown off trying to follow these examples. This PR updates the documentation to match what worked for us!

### CODE REVIEW

File: docs/manual/src/udl/callback_interfaces.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ which may use it.
 In UDL:
 
 ```webidl
-object Authenticator {
+interface Authenticator {
```
Comment (bendk) [inline docs/manual/src/udl/callback_interfaces.md:50]: I think this should actually be `callback interface Authenticator {`

**Review by bendk — CHANGES_REQUESTED** [review]
Looks good and sorry that the docs caused issues for you.

I had one small suggestion to improve them even more.

File: docs/manual/src/udl/callback_interfaces.md (hunk 1 above)
Comment (quad) [reply docs/manual/src/udl/callback_interfaces.md:50]:
```suggestion
callback interface Authenticator {
```

Comment (quad) [reply docs/manual/src/udl/callback_interfaces.md:50]: Are you sure? As I read it, the `Authenticator` is an interface that _uses_ the `callback interface Keychain`.

Comment (bendk) [reply docs/manual/src/udl/callback_interfaces.md:50]: Ahh yes, my mistake.  Merging now.

**Review by bendk — APPROVED** [review]

**Pull request merged**